docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
I for one is in favor of waiting at lease next patch release of Docker before using new features.